server/src/services/item.service.ts

```diff
@@ -61,6 +77,25 @@ class ItemService {
       limit,
     });
   }
+
+  async autoCompleteKeyword(keyword: string): Promise<string[]> {
+    const result = await this._searchToElasticSearch(keyword);
+    const titles = result.hits.hits.map((hit) => {
+      const { title } = hit._source as Item;
+      return title;
+    });
+
+    return titles;
+  }
```

Comment:
```suggestion
  async autoCompleteKeyword(keyword: string): Promise<string[]> {
    const result = await this._searchToElasticSearch(keyword);
    return result.map((hit) => hit._source.title);
  }
```

`esClient.search<Item>`으로 제네릭 타입을 지정하고,
`result.hits.hits`를 반환했다면 아예 이렇게 줄일 수도 있겠어요 ☺️

Reply:
좋습니다 👍